install/parse-cli.sh

```diff
@@ -4,14 +4,20 @@ show_help() {
   cat <<EOF
 Usage: $0 [options]
 
-Install Sentry with `docker compose`.
+Install Sentry with \`docker compose\`.
```

Comment:
Oops. Backticks spawn a subshell here, need to escape them. Regression in #1179.